browser/profiles/brave_profile_manager_browsertest.cc

```diff
@@ -243,225 +183,13 @@ IN_PROC_BROWSER_TEST_F(BraveProfileManagerTest,
 #endif
 }
 
-#if BUILDFLAG(ENABLE_TOR)
-IN_PROC_BROWSER_TEST_F(BraveProfileManagerTest,
-                       SwitchToTorProfileShareBookmarks) {
-  ScopedTorLaunchPreventerForTest prevent_tor_process;
-  ProfileManager* profile_manager = g_browser_process->profile_manager();
-  ASSERT_TRUE(profile_manager);
-  Profile* parent_profile = ProfileManager::GetActiveUserProfile();
-
-  // Add a bookmark in parent profile.
-  const base::string16 title(base::ASCIIToUTF16("Test"));
-  const GURL url1("https://www.test1.com");
-  bookmarks::BookmarkModel* parent_bookmark_model =
-      BookmarkModelFactory::GetForBrowserContext(parent_profile);
-  bookmarks::test::WaitForBookmarkModelToLoad(parent_bookmark_model);
-  const bookmarks::BookmarkNode* root =
-      parent_bookmark_model->bookmark_bar_node();
-  const bookmarks::BookmarkNode* new_node1 =
-      parent_bookmark_model->AddURL(root, 0, title, url1);
-
-  Profile* tor_profile = SwitchToTorProfile();
-  ASSERT_TRUE(brave::IsTorProfile(tor_profile));
-  EXPECT_TRUE(tor_profile->IsOffTheRecord());
-  EXPECT_EQ(brave::GetParentProfile(tor_profile), parent_profile);
-
-  // Check if the same node is in Tor profile since we share the bookmark
-  // service between Tor profile and its parent profile.
-  bookmarks::BookmarkModel* tor_bookmark_model =
-      BookmarkModelFactory::GetForBrowserContext(tor_profile);
-  bookmarks::test::WaitForBookmarkModelToLoad(tor_bookmark_model);
-  EXPECT_EQ(tor_bookmark_model->GetMostRecentlyAddedUserNodeForURL(url1),
-            new_node1);
-
-  // Add a new bookmark in parent profile again and check if it would show up
-  // in Tor profile.
-  const GURL url2("https://www.test2.com");
-  const bookmarks::BookmarkNode* new_node2 =
-      parent_bookmark_model->AddURL(root, 0, title, url2);
-  EXPECT_EQ(tor_bookmark_model->GetMostRecentlyAddedUserNodeForURL(url2),
-            new_node2);
-
-  // Add a new bookmark through tor profile and check if it would show up in
-  // its parent profile.
-  const GURL url3("https://www.test3.com");
-  const bookmarks::BookmarkNode* tor_root =
-      tor_bookmark_model->bookmark_bar_node();
-  const bookmarks::BookmarkNode* new_node3 =
-      tor_bookmark_model->AddURL(tor_root, 0, title, url3);
-  EXPECT_EQ(parent_bookmark_model->GetMostRecentlyAddedUserNodeForURL(url3),
-            new_node3);
-}
-
 IN_PROC_BROWSER_TEST_F(BraveProfileManagerTest,
-                       SwitchToTorProfileExcludeServices) {
-  ScopedTorLaunchPreventerForTest prevent_tor_process;
-
+                       GetLastUsedProfileName) {
+  g_browser_process->local_state()->SetString(
+      prefs::kProfileLastUsed, base::FilePath(tor::kTorProfileDir).value());
```

Comment:
base::FilePath::value() will return a wstring on Windows.

Reply:
:+1: will use `AsUTF8Unsafe()`

Reply:
fixed in 1acf6fe166dfd865f8df5ebc59c4d67de0513e85